org.openhab.binding.zigbee/ESH-INF/thing/channels.xml

```diff
@@ -136,6 +136,24 @@
         <category>Sensor</category>
         <state readOnly="true" />
     </channel-type>
+    
+    <!-- IAS Movement Sensor Channel -->
+    <channel-type id="ias_movement">
+        <item-type>Switch</item-type>
+        <label>Movement Sensor</label>
+        <description>Movement Sensor Alarm</description>
+        <category>Sensor</category>
```

Comment:
I wonder why `ias_water` has category `Sensor` though the description mentions "Alarm" and `ias_tamper` has category `Alarm`.

So IMHO all alarm related channels should have category `Alarm`. WDYT?

Reply:
Good question; there are other IAS-based channels that also have a category that is not `Alarm`, and where I think that they should keep that category, viz the motion detection channels.

Regarding the water alarm, I don't know whether `Sensor` or `Alarm` is better. But I think that this is out of scope of this PR.

Reply:
I am fine with mixing tags, I was just wondering whether all these channels with `Alarm` in their description should have the `Alarm` category. But lets leave it as a sensor now.